### ISSUE TITLE
RtUser.friends() not implemented

### ISSUE THREAD

**Issue by dmzaytsev** [yegor256/netbout#612]
`com.netbout.client.RtUser.friends(String text)` not implemented yet but has to be. 
Let's implement it and don't forget about the unit test

**Comment by dmzaytsev** [comment]
@karato it's a valid bug

**Comment by dmzaytsev** [comment]
@karato please set milestone to 3.0

**Comment by karato** [comment]
> @karato it's a valid bug

@dmzaytsev I tagged this as "bug"

**Comment by karato** [comment]
@dmzaytsev milestone set to `3.1` (correct me if I am wrong)

**Comment by karato** [comment]
> @karato please set milestone to 3.0

@dmzaytsev milestone set to `3.0`, thanks

**Comment by karato** [comment]
@dmzaytsev thank you for the ticket reported, I topped your account for 15 mins, transaction `57693110`

**Comment by karato** [comment]
@ekondrashev this is your task

**Comment by ekondrashev** [comment]
@karato this one is blocked by #622

**Comment by karato** [comment]
> @karato this one is blocked by #622 

@ekondrashev sure, let's wait for #622

**Comment by dmzaytsev** [comment]
@karato this is postponed

**Comment by karato** [comment]
> @karato this is postponed

@dmzaytsev thanks, I added "postponed" label

**Comment by karato** [comment]
> @karato this is postponed

@dmzaytsev OK, I will try to assign someone else

**Comment by dmzaytsev** [comment]
@karato set milestone 3.1

**Comment by karato** [comment]
> @karato set milestone 3.1

@dmzaytsev the milestone here is 3.1
